app/assets/javascripts/protocol_form.js.coffee

```diff
@@ -185,8 +185,13 @@ $(document).ready ->
     return
 
   $(document).on 'change', restrict_sending_dropdown, (e) ->
-    data = { ans1: $(certificate_of_confidence_dropdown).val(), ans2: $(higher_level_of_privacy_dropdown).val(), ans3: $(epic_inbasket_dropdown).val(), ans4: $(research_active_dropdown).val(), ans5: $(restrict_sending_dropdown).val() }
-    determine_study_type(data)
+    new_value = $(e.target).val()
+    if new_value != ''
+      data = { ans1: $(certificate_of_confidence_dropdown).val(), ans2: $(higher_level_of_privacy_dropdown).val(), ans3: $(epic_inbasket_dropdown).val(), ans4: $(research_active_dropdown).val(), ans5: $(restrict_sending_dropdown).val() }
```

Comment:
I would use Coffeescript's syntax for Objects here. See "Object literals & array definition" at https://arcturo.github.io/library/coffeescript/02_syntax.html